File: program/Primes/Primes.rb
[100].flat_map{|n|
	(1..Math.sqrt(n).to_i)
	.flat_map{|x|(2..n/x)
		.map{|y|x*y}}}
.group_by{|x|x}
.filter{|k,v|v.size==1}
.map{|k,v|k}
.each{|x|puts x}

File: program/Primes/Primes.kt
fun main() {
    listOf(100)
    .flatMap{n->(1..kotlin.math.sqrt(n.toDouble()).toInt())
        .flatMap{x->(2..n/x).map{y->x*y}}}
    .groupBy{it}
    .filter{it.value.count()==1}
    .map{it.key}
    .forEach(::println);
}
